kolibri/core/content/migrations/0017_auto_20190415_1855.py

```diff
@@ -1,19 +0,0 @@
-# -*- coding: utf-8 -*-
-# Generated by Django 1.11.20 on 2019-04-15 10:55
-from __future__ import unicode_literals
-
-from django.db import migrations
-from django.db import models
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [("content", "0016_auto_20190124_1639")]
-
-    operations = [
-        migrations.AlterField(
-            model_name="channelmetadata",
-            name="published_size",
-            field=models.BigIntegerField(blank=True, default=0, null=True),
```

Comment:
@indirectlylit that makes sense. I cannot recall exactly why I deleted this migration. Should I leave it deleted in this instance since the repo includes a migration file that replaces this one?

Reply:
The other thing to bear in mind is that other migrations might be added which need to be previous to this in the migration tree (e.g. https://github.com/learningequality/kolibri/pull/5497/files#diff-21bfbbd534842850877c325a17b3d32a), so the correct migration (particularly on the develop branch) can end up being a bit of a moving target.

The best thing to do in this case would be to reinstate the previous `0017` migration, then create a new migration just for what you are doing.

If the migration I note above ends up getting merged before your PR, then we will need to recreate the migration to take that into account. Otherwise it can get sorted out by the person who merges it into develop.